src/design-system/components/Badge/Badge.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+
+import Badge from './Badge';
+
+export default {
+  title: 'Badge',
```

Comment:
```suggestion
  title: 'Components/Badge',
```

If we change the title to include that prefix, I think it'll fit nicely under the existing components section.

Perhaps we should make a new section though, to keep new components separate from the Emotion ones?

Reply:
We could make a new section, I think that makes sense and then slowly move stuff from components away to it, would be a nice visual way to see the work done/to be done in Storybook, maybe something like `Design-System/Badge`?

Reply:
I think that sounds reasonable :)